worker/src/features/batchExport/handleBatchExportJob.ts

```diff
@@ -0,0 +1,169 @@
+import { pipeline } from "stream";
+
+import {
+  BatchExportFileFormat,
+  BatchExportJobType,
+  BatchExportQuerySchema,
+  BatchExportStatus,
+  exportOptions,
+  FilterCondition,
+  getSessionTableSQL,
+} from "@langfuse/shared";
+import { prisma } from "@langfuse/shared/src/db";
+import {
+  DatabaseReadStream,
+  S3StorageService,
+  sendBatchExportSuccessEmail,
+  streamTransformations,
+} from "@langfuse/shared/src/server";
+
+import { env } from "../../env";
+import logger from "../../logger";
+
+export const handleBatchExportJob = async (
+  batchExportJob: BatchExportJobType
+) => {
+  const { projectId, batchExportId } = batchExportJob;
```

Comment:
Just to confirm. The security concept is as follows:
- User provides projectId via trpc route
- Only this project id is inserted into the database and sent to our events endpoint
- projectId is reade from the db here and passed on to getSessionTableSQL which is filtering by projectId